core/.coveragerc

```diff
@@ -0,0 +1,16 @@
+[run]
+branch = True
+source = acpgpy
+parallel = True
+
+[paths]
+source =
+    acpgpy
+   .tox/*/lib/python*/site-packages/acpgpy
+   .tox/*/lib/site-packages/acpgpy
+   .tox/pypy*/site-packages/acpgpy
+
+[report]
+omit =
+ */python?.?/*
+ *__init__*
```

Comment:
the "acpgpy" references look wrong now.  Also i'd prefer to discuss/add coverage and other testing in a separate PR because this PR is already quite big ...